index.md

```diff
@@ -24,13 +24,13 @@ A converter and validator is available at [http://server1.nlp.insight-centre.org
 XML
 ---
 
-The XML is specified by the following [DTD](WN-LMF-1.3.dtd). An example is 
+The XML is specified by the following [DTD](WN-LMF-1.4.dtd). An example is 
 given [here](https://github.com/globalwordnet/schemas/blob/master/example.xml):
 
 The first three lines must always be as follows:
 
     <?xml version="1.0" encoding="UTF-8"?>
-    <!DOCTYPE LexicalResource SYSTEM "http://globalwordnet.github.io/schemas/WN-LMF-1.3.dtd">
+    <!DOCTYPE LexicalResource SYSTEM "http://globalwordnet.github.io/schemas/WN-LMF-1.4.dtd">
```

Comment:
The `http` URL still works, but should we change this to `https`? Do any validators have trouble with `https` links?

Reply:
Not sure it matters either way. I doubt any modern tool has problem with https, but also the URL with http will still work.

Reply:
Ok let's leave it for now. I was just concerned about browsers blocking `http` links or us serving [mixed http--https content](https://docs.github.com/en/pages/getting-started-with-github-pages/securing-your-github-pages-site-with-https#resolving-problems-with-mixed-content).